librocksdb_sys/crocksdb/c.cc

```diff
@@ -666,6 +683,42 @@ crocksdb_t* crocksdb_open_column_families(
   return result;
 }
 
+crocksdb_t* crocksdb_open_column_families_with_ttl(
+    const crocksdb_options_t* db_options,
+    const char* name,
+    int num_column_families,
+    const char** column_family_names,
+    const crocksdb_options_t** column_family_options,
+    crocksdb_column_family_handle_t** column_family_handles,
+    const int32_t* ttl_array,
```

Comment:
Please put ttl_array before column_family_handles, cause it's a input parameter.

FYI: https://google.github.io/styleguide/cppguide.html#Output_Parameters